lib/app/utils.dart

```diff
@@ -32,8 +32,12 @@ String formatFileSize(int bytes) {
   return '${(bytes / power).toStringAsFixed(index == 0 ? 0 : 1)} ${units[index]}';
 }
 
-/// Converts a DateTime to a string.
-String dateTimeToString(DateTime dt) => DateFormat.yMMMd().format(dt);
+/// Converts a [DateTime] to a [String].
+String dateTimeToString(DateTime dt) => DateFormat.MMMd().format(dt);
+
+/// Converts a [String] to a [DateTime].
+DateTime parseDateTime(String string) =>
+    DateTime.parse(string.replaceAll('T', ' ').replaceAll('Z', ''));
```

Comment:
Out of curiosity: why are "T" and "Z" removed? These are standardized by ISO 8601.

But: We should use the DateTime handling of our calendar module (especially https://github.com/schul-cloud/schulcloud-flutter/blob/master/lib/calendar/data.dart#L143-L160) and use `Instant`s to correct the wrong server times.

Reply:
I'm not entirely sure anymore why the `T` and `Z` strings get replaced.
As issue #117 already entails revising how `DateTime`s are handled throughout the project, I'd argue we rewrite this at a later `DateTime` (sorry for the pun).